CMakeLists.txt

```diff
@@ -89,7 +89,6 @@ CHECK_INCLUDE_FILES("fcgios.h" FASTCGI_FASTCGIOS_DIR)
 CHECK_INCLUDE_FILES("fcgi_stdio.h" HAVE_FASTCGI_STDIO_H)
 CHECK_INCLUDE_FILES("openssl/ssl.h" HAVE_SSL_H)
 CHECK_INCLUDE_FILES("keyutils.h" HAVE_KEYUTILS_H)
-CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
```

Comment:
@cbodley i guess you are removing the last place where `HAVE_EXECINFO_H` is defined?